spec/models/mou_signature_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+
+RSpec.describe MouSignature, type: :model do
+  let(:organisation) { create(:organisation) }
+  let(:user) { create(:user, organisation:) }
+  let(:mou_signature) do
+    described_class.create!(
+      user:,
+      organisation:,
+      agreed_at: Time.zone.now,
+    )
+  end
+
+  it "setting user to nil triggers a DB constraint" do
+    mou_signature.user = nil
+    expect { mou_signature.save(validate: false) }.to raise_error(ActiveRecord::StatementInvalid).with_message(/null value in column "user_id"/)
+    # expect { mou_signature.update_column(:user_id, nil) }.to raise_error(ActiveRecord::StatementInvalid).with_message(/null value in column "user_id"/)
```

Comment:
Do we still need this commented out line?

Reply:
We did not, good spot!